packages/datadog-plugin-aws-sdk/src/aws_helpers.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+const awsHelpers = {
+  wrapCallback (tracer, span, done, parent, config) {
+    return function (err, response) {
+      // "this" should refer to response object https://github.com/aws/aws-sdk-js/issues/781#issuecomment-156250427
+      awsHelpers.addAdditionalTags(span, this)
+
+      config.hooks.addCustomTags(span, this.request.params)
+      awsHelpers.finish(span, err, config)
+
+      if (typeof done === 'function') {
+        tracer.scope().activate(parent, () => {
```

Comment:
should i be using tracer.scope.bind here? or more generally?

Reply:
i went ahead and updated both this section, and other relevant areas in `base.js`, to try to use `scope().bind()` instead of `scope().activate`. I _think_ that's the proper way to do instrumentation in this case? lmk what you think.

Reply:
In general, `bind` would be used when you return the function without calling it, and `activate` is used when you call the function immediately. `bind` does work in both cases though, but it creates an additional function in the process compared to `activate`.